src/internal/operations/backup.go

```diff
@@ -632,23 +633,98 @@ func getNewPathRefs(
 	return newPath, newLoc, updated, nil
 }
 
-func mergeDetails(
+// mergeDetailsFromAssistBackups merges the deets from the assist backups into
+// the current deets.
+func mergeDetailsFromAssistBackups(
 	ctx context.Context,
 	detailsStore streamstore.Streamer,
-	backups []kopia.BackupEntry,
-	dataFromBackup kopia.DetailsMergeInfoer,
 	deets *details.Builder,
-	writeStats *kopia.BackupStats,
+	assistBackups []kopia.BackupEntry,
 	serviceType path.ServiceType,
```

Comment:
nit: could you remove this please? It appears to be unused (shame on me for not noticing that when I wrote this)
```suggestion
```